### ISSUE TITLE
sequentialDownload command for transmission-remote | Bug with -G option

### ISSUE THREAD

**Issue by emrakyz** [transmission/transmission#6658]
### Sequential Downloading on transmission-remote
I have analyzed the source code and seen **sequentialDownload** is implemented, is a boolean value and we also see the sequential status with `transmission-remote -t [0-9]* --info` command.

On the other hand, I can directly call `transmission-cli -seq` command to start a torrent with sequential downloading enabled.

Nevertheless, I have found no solution for enabling or disabling sequential downloading through `transmission-remote` command itself in order to write scripts around. I am not sure if the direct communication through this command is implemented or not. Can you clarify?

### `-G (--no-get)` option does not work properly
This is probably a duplicate and I understand the fact that torrent files can share the same piece and therefore files (or some parts of them) are downloaded regardless but I guess, a function can be implemented to remove those files later automatically.

### Which application of Transmission?

transmission-remote

### Which version of Transmission?

4.1.0-dev (1855cdb731)

**Comment by tearfur** [comment]
> I am not sure if the direct communication through this command is implemented or not. Can you clarify?

You are right, there is no option to specify sequential download through `transmission-remote` at the moment.

> This is probably a duplicate and I understand the fact that torrent files can share the same piece and therefore files (or some parts of them) are downloaded regardless but I guess, a function can be implemented to remove those files later automatically.

If we delete the partial files that were specified by `-G`, then we cannot seed the pieces that contains those files later on. So that is not an option AFAICT.

**Comment by emrakyz** [comment]
Thanks! @tearfur  These answer my questions.

But yet, can't we keep those files as `tmp` so they will be deleted if the user removes the torrent completely from the list. So, with this way, seeding won't be affected and the download directory won't be polluted.

Right now, this creates a huge problem in my opinion. Let's say we download a torrent with 500 different files and nested directory structures. Normally when we open the download directory, we should see the selected files but now there are hundreds of unrelated and incomplete files so we need to search for what we actually download and then, select & remove all of the others for every single torrent file.

**Comment by andreygursky** [comment]
> a function can be implemented to remove those files later automatically

```-g/-G``` are used for select/deselect, thus ```-G``` should not trigger data removal.

Additionally to ```-r``` and ```-R``` a new delete function could be implemented, that would delete files with data but without any wanted piece. An advanced mode would be to purge hole in existing files, where only 1 partial piece at the beginning and/or 1 partial piece at the end is wanted, thus freeing the whole space (except of only few KB/MB).

**Comment by tearfur** [comment]
> can't we keep those files as `tmp` so they will be deleted if the user removes the torrent completely from the list. So, with this way, seeding won't be affected and the download directory won't be polluted.

@emrakyz The incomplete directory (`incomplete-dir` in settings.json) does pretty much what you described.

**Comment by Mehdinouira** [comment]
question about running command 
transmission-cli -seq episode 1,2,3,4   chuck.srt     ???
note. im using mac
